src/test/java/hudson/plugins/claim/ClaimTest.java

```diff
@@ -274,6 +276,14 @@ public void claimTestShouldGiveProperURL() {
         assertEquals("wrong url who would not link to test",
                 "job/x/1/testReport/junit/assembly/classTest/unitTest", acti.getUrl());
     }
+    @Test
+    public void claimTestShouldGiveProperURL() {
```

Comment:
Could you elaborate a little bit to how this is related to the proposed change ?